cmd/cstor-pool-mgmt/controller/start-controller/controller.go

```diff
@@ -129,6 +134,15 @@ func StartControllers(kubeconfig string) {
 		}
 		wg.Done()
 	}()
+
+	wg.Add(NumRoutinesThatFollow)
+	// Run controller for BackupCStor.
+	go func() {
+		if err = backupController.Run(NumThreads, stopCh); err != nil {
```

Comment:
is backup multi-threaded from API server and pool-mgmt side?

Reply:
It is single-threaded from pool-mgmt side. API server is also single-threaded. .